src/test/java/plugins/LdapPluginTest.java

```diff
@@ -70,6 +70,34 @@ private LdapDetails createDefaultsWithoutManagerCred(LdapContainer ldapContainer
     }
 
     @Test
+    @WithPlugins("ldap@1.15")
```

Comment:
I think you need to fix [this](https://github.com/jenkinsci/acceptance-test-harness/blob/bb8772bccbd8f882761df91d102be2b7fe343583/src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java#L121) first, or simply expect 1.15 for the whole test suite.

(ATH in its normal run mode will always download the latest version of the plugin, so a version annotation is only useful then for documentation or blocking a test from running until a snapshot is released. There are less commonly used modes whereby you can test against a live server or a WAR bundling plugins, in which case a newer version annotation can force a test to be skipped or an update applied IIRC. But in practice we do not much care about testing historical plugin versions, any more than we expect historical revisions of ATH to continue to pass today.)